src/utilities/test-utils.ts

```diff
@@ -35,6 +35,22 @@ const _randomValue = <TValue = any>(obj: any): TValue => obj[_randomKey(obj)];
  */
 const _randomWord = (): string => faker.random.word().split(" ")[0];
 
+/**
+ * Generates random object
+ */
+const _randomObject = (keyCount?: number) => {
```

Comment:
Nitpick - would you mind alphabetizing this function inside this file?

Reply:
Done